### ISSUE TITLE
Reactivity for pan not working

### ISSUE THREAD

**Issue by mexx42** [HassaanAkbar/vue-zoomable#16]
Hi There, thanks for your great job, however i found out that reactivity on pan is not working. 

here is the patch (which i assume doesnt need a PR ;):

```
diff --git a/src/components/VueZoomable.vue b/src/components/VueZoomable.vue
index a9e1654..d252f87 100644
--- a/src/components/VueZoomable.vue
+++ b/src/components/VueZoomable.vue
@@ -130,9 +130,10 @@ watch(
   () => {
     if (props.pan) {
       pan.value.x = props.pan.x;
-      pan.value.x = props.pan.y;
+      pan.value.y = props.pan.y;
     }
-  }
+  },
+  {deep: true}
 );
 
 function emit(name: string, event: ZoomableEvent) {
```
also the line containing:
```
import { eventNames } from 'process';
```
is not used, but thats just cosmetics ... 

can you please fix this and make a new version.

thank you very much,
mexx.

**Comment by ehassaan** [comment]
Hi Mexx. Thanks for the issue and patch. I will try and update.

**Comment by ehassaan** [comment]
Issue is fixed in v1.1.8. 
https://hassaanakbar.github.io/vue-zoomable/demos/reactivity-demo.html
